packages/react-native-reanimated/android/src/main/cpp/worklets/android/WorkletsModule.h

```diff
@@ -46,19 +46,19 @@ class WorkletsModule : public jni::HybridClass<WorkletsModule> {
   }
 
  private:
-  friend HybridBase;
-  jni::global_ref<WorkletsModule::javaobject> javaPart_;
-  jsi::Runtime *rnRuntime_;
-  std::shared_ptr<WorkletsModuleProxy> workletsModuleProxy_;
-
   explicit WorkletsModule(
-      jni::alias_ref<WorkletsModule::jhybridobject> jThis,
       jsi::Runtime *rnRuntime,
       const std::string &valueUnpackerCode,
       jni::alias_ref<JavaMessageQueueThread::javaobject> messageQueueThread,
       const std::shared_ptr<facebook::react::CallInvoker> &jsCallInvoker,
       const std::shared_ptr<worklets::JSScheduler> &jsScheduler,
       const std::shared_ptr<UIScheduler> &uiScheduler);
+
+  void invalidateCpp();
+
+  friend HybridBase;
+  jsi::Runtime *rnRuntime_;
```

Comment:
If possible, let's store runtime as a reference.
```suggestion
  jsi::Runtime &rnRuntime_;
```

Reply:
https://github.com/software-mansion/react-native-reanimated/pull/6850#discussion_r1908596428